src/components/AboutUs/TimelineSection.js

```diff
@@ -188,45 +171,38 @@ export default function TimelineSection() {
     return (
       <Grid container justify="center" alignItems="center">
         <Grid item xs={12} className={classes.center}>
-          {/* <img alt="title decoration left" src={titleLeft} /> */}
           <TitleLeft />
           <span className={classes.title}>See how we’ve grown!</span>
           <TitleRight />
-          {/* <img alt="title decoration right" src={titleRight} /> */}
         </Grid>
         <Grid className={classes.gridContainer} container item xs={12}>
           <Grid item xs={2} md={3} lg={3} className={classes.center}>
             <span className={classes.yearTag}>2017</span>
             <Pin className={classes.pinStyle} />
-            {/* <img className={classes.pinStyle} alt="pin" src={pin} /> */}
           </Grid>
           <Grid container item xs={10} md={9} lg={9}>
             <span className={classes.yearTagSm}>2017</span>
             <span className={classes.eventTitle}>
               Research and program development
             </span>
-            {/* <LeftBlob style = {StylesProvider.overlay/> */}
             <p className={classes.eventContent}>
               Toronto Tech Mentoring was born out of the Civic Tech Toronto
               space. Civic Tech is a movement that focuses on developing and
               harnessing technology for the betterment of civic life.
             </p>
             <div>
-              <Pic1
+              <img
                 className={`${classes.firstPic} ${classes.pic}`}
-                // src={pic1}
+                src={pic1}
               />
-              <Pic1 className={classes.pic}/>
-              {/* <img className={classes.bg} alt="purple bubble" src={bg3} /> */}
+              <img className={classes.pic} src={pic1}/>
```

Comment:
lets also add "alt" text - and add a to-do item in "accessibility" card in trello to know what alt text values should be